modules/KIWILinuxRC.sh

```diff
@@ -6967,6 +6967,21 @@ function mountSystem {
     return $retval
 }
 #======================================
+# mountLivecd
+#--------------------------------------
+function mountLivecd {
+    if [ -n $TORAM ]; then
+        mkdir -p ${LIVECD}R $LIVECD && eval mount $cdopt $biosBootDevice ${LIVECD}R
+        SIZE="$(/usr/bin/du -s /${LIVECD}R | /usr/bin/gawk '{print int($1*1.1)}')"
+        test -n $SIZE || SIZE="800000"
+        /bin/mount -t tmpfs -o size=${SIZE}k tmpfs $LIVECD
+        cp -r ${LIVECD}R/* $LIVECD/
```

Comment:
Instead of cp please use rsync, that avoids problems with file permissions. rsync is available in the initrd